plugins/source/oracledb/client/transformer.go

```diff
@@ -135,3 +140,41 @@ func (*Transformer) TransformTextArray(v *schema.TextArray) any {
 
 	return v.String()
 }
+
+func (*Transformer) RecordToCQTypes(table *schema.Table, record arrow.Record) (schema.CQTypes, error) {
```

Comment:
Can we just bring back the old `GenData` for this? this will make the migration simpler for now. once we migrate sources to arrow this would also be gone.